apisix/plugins/limit-count.lua

```diff
@@ -70,11 +74,36 @@ local schema = {
                             type = "string", minLength = 0,
                         },
                         redis_timeout = {
-                            type = "integer", minimum = 1,
-                            default = 1000,
+                            type = "integer", minimum = 1, default = 1000,
                         },
                     },
                     required = {"redis_host"},
+                },
+                {
+                    properties = {
+                        policy = {
+                            enum = {"redis-cluster"},
+                        },
+                        redis_serv_list = {
```

Comment:
`redis_serv_list` is not a good name, how about `nodes`?

Reply:
updated to redis_cluster_nodes.